src/System.Threading.Thread/tests/ThreadTests.cs

```diff
@@ -171,6 +171,7 @@ public static void ApartmentState_AttributePresent(string AppName, string mode)
         }
 
         [Fact]
+        [ActiveIssue(20766,TargetFrameworkMonikers.UapAot)]
```

Comment:
Should we change it to Uap ? because for ```ApartmentState_AttributePresent``` , I tried disabling it just on UapAot but still the build UWP CoreCLR x64 Debug Build this was failing.

Reply:
I can't explain that since it's passing in this CI job.